.circleci/config.yml

```diff
@@ -68,11 +68,10 @@ jobs:
             pipenv run ansible-galaxy install -r requirements.yml
             cp .circleci/.vault ~/.vault;
             chmod +x ~/.vault
-            pipenv run ansible-playbook -i inventory/prod/hosts playbook.yml --tags "jumphost,role::airflow::dags" --vault-password-file=~/.vault -e 'ansible_ssh_port=9229'
-
+            pipenv run ansible-playbook -i inventory/prod playbook.yml --tags "jumphost,role::airflow::dags" --vault-password-file=~/.vault -e 'ansible_ssh_port=9229' -e manifold_dags_branch=$CIRCLE_TAG
```

Comment:
Does it need to be the name of the repo?  The full name is manifold_airflow_dags.

Reply:
@sensei100 it needs to match what it's called in the playbook: https://github.com/tulibraries/ansible-playbook-airflow/blob/aad20ca8122bddee6003b2e8e86f438d52c290b1/inventory/qa/group_vars/all.yml#L118